lib/devices_list/devices_list_view.dart

```diff
@@ -17,14 +19,19 @@ class DevicesListScreen extends StatefulWidget {
 class DeviceListScreenState extends State<DevicesListScreen> {
 
   DevicesBloc _devicesBloc;
+  StreamSubscription _appStateSubscription;
+
 
   @override
-  void initState() {
-    super.initState();
+  void didChangeDependencies() {
+    super.didChangeDependencies();
     _devicesBloc = DevicesBlocProvider.of(context);
```

Comment:
Czy nie powinniśmy wywołać `dispose` na `_devicesBloc` zanim nadpiszemy go nowym obiektem?

Reply:
w drugim PR pracuję nad refactorem inicjowanie i sprzątania bo jak się okazuje działa to źle i wymaga to większego przerobienia więc zostawmy to proszę tutaj na ten moment.